docs/sql-programming-guide.md

```diff
@@ -1061,7 +1061,7 @@ the following case-sensitive options:
   </tr>
 
   <tr>
-    <td><code>partitionColumn, lowerBound, upperBound, numPartitions</code></td>
```

Comment:
In the [read path](https://github.com/dongjoon-hyun/spark/blob/48b6d2513ddebcf365999c8567a7e6dbcc5a3bb2/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala#L92-L93), they are still related.